src/vivarium_gates_child_iv_iron/tools/cli.py

```diff
@@ -41,6 +41,10 @@ def make_artifacts(location: str, output_dir: str, append: bool, replace_keys: T
 
 @click.command()
 @click.argument('output_file', type=click.Path(exists=True))
+@click.option('-is', '--include-seeds',
+              default=False,
```

Comment:
Avoid `-is` as single-dash flags are but compound args (e.g., `rm -rf`) having two options, recursive and force.

https://www.cs.princeton.edu/courses/archive/spr96/cs333/java/tutorial/java/cmdLineArgs/conventions.html